src/components/CopyButton/index.tsx

```diff
@@ -20,6 +20,12 @@ export const CopyButton = (props: CopyButtonProps) => {
         onClick={() => {
           copy(textToCopy)
         }}
+        style={{
```

Comment:
Please use tailwindcss classes.

Reply:
Tailwind has a pretty steep learning curve. 

I know this project chose to go with it, and it's unlikely that it will change, but as a contributor who's never dealt with it, I find it really discouraging having to learn that thing as well.

Since styling accounts to <10% of this PR I hoped someone else may help with it 🤞🏻

Reply:
Will do then.